CONTRIBUTING.md

```diff
@@ -219,6 +219,45 @@ name.
 </record>
 ```
 
+### External dependencies
+
+#### `__openerp__.py`
+If your module use extras dependencies of python or binaries you should add to `__openerp__py` file the section `external_dependencies`.
+
+```python
+{
+    'name': 'Module of example',
+    'depends': ['base'],
+    ...
+    'external_dependencies': {
+        'bin': [
+            'external_dependency_binary_1',
+            'external_dependency_binary_2',
+            ...
+            'external_dependency_binary_N',
+        ],
+        'python': [
+            'external_dependency_python_1',
+            'external_dependency_python_2',
+            ...
+            'external_dependency_python_N',
+        ],
+    },
+    ...
+    'installable': True,
+}
+```
+
+#### ImportError
+In python files where you use a `import external_dependency_python_N` you will need to add a `try-except` with a info log.
+
+```python
+try:
```

Comment:
No, my option is to include the external dependency:

https://github.com/OCA/l10n-spain/blob/8.0/l10n_es_aeat/__openerp__.py#L48

and the import inside the method which needs it: 

https://github.com/OCA/l10n-spain/blob/8.0/l10n_es_aeat/wizard/export_to_boe.py#L58

This way, you don't receive any annoying log line with uninstalled modules, and when installing, module manager will check if you have the lib. You won't need either to make any check here, because when using method, you already assure that the library is available.